media-manage-state/src/main/scala/models/shards/FileShards.scala

```diff
@@ -4,27 +4,12 @@ import java.util.UUID
 
 import akka.actor.typed.ActorSystem
 import akka.actor.typed.ActorRef
-
-import akka.persistence.typed.scaladsl.{ Effect, ReplyEffect }
-
+import akka.persistence.typed.scaladsl.{Effect, ReplyEffect}
 import media.state.media.MediaConverter
-import media.state.models.actors.FileActor.{
-  AddFile,
-  CompressFile,
-  Config,
-  ConvertFile,
-  ConvertedFile,
-  FileAdded,
-  FileJournal,
-  GetFile,
-  State,
-  UpdateStatus,
-  UpdatedStatus
-}
+import media.state.models.actors.FileActor.{AddFile, CompressFile, Config, ConvertFile, ConvertedFile, FileAdded, FileJournal, GetFile, PlayFile, State, UpdateStatus, UpdatedStatus}
```

Comment:
I'd advise against an import line (or any source code line) that is too long. Consider breaking it up into multiple lines:

```
import media.state.models.actors.FileActor. {
  AddFile,
  CompressFile,
  ... // more imports here
}
```

Or just import everything in this case since there's too many imports: `import media.state.models.actors.FileActor._`

Reply:
resolved at [this commit](https://github.com/Try-Parser/MediaFileEncoder/pull/20/commits/7ee43c7d7d536ad1d132d52ab9fa2a8e29232ca5)